acct-user/dbmail/dbmail-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="user for dbmail daemons"
+ACCT_USER_ID=277
+ACCT_USER_HOME="/var/lib/dbmail"
```

Comment:
Is this really needed? Please see http://michael.orlitzky.com/articles/gentoo_glep81_user_package_guidelines.xhtml#the-home-directory for reference.

Reply:
Dbmail supports different database backends; one of them is sqlite.  Sqlite database location should default to dbmail's home directory.
Other database backends have no need to write to /var/lib/dbmail.